prov/verbs/src/verbs_info.c

```diff
@@ -975,9 +975,6 @@ static int fi_ibv_fill_addr(struct rdma_addrinfo *rai, struct fi_info **info,
 	struct sockaddr *local_addr;
 	int ret;
 
-	if (rai->ai_src_addr)
-		goto rai_to_fi;
-
```

Comment:
This would break the case when the server has to listen on a wildcard IP address but a known port.